model/registry/file.yaml

```diff
@@ -4,12 +4,56 @@ groups:
     type: attribute_group
     brief: "Describes file attributes."
     attributes:
+      - id: accessed
+        type: string
+        brief: >
+          Time when the file was last accessed, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: attributes
+        type: string[]
+        brief: >
+          Array of file attributes.
+        note: >
+          Attributes names will vary by platform. Here’s a non-exhaustive list of values that are expected in this
+          field: archive, compressed, directory, encrypted, execute, hidden, read, readonly, system, write.
+        stability: experimental
+        examples: ['readonly', 'hidden']
+      - id: created
+        type: string
+        brief: >
+          Time when the file was created, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: ctime
+        type: string
+        brief: >
+          Time when the file attributes or metadata was last changed, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems. Note that changes to the
+          file content will update `mtime`. This implies `ctime` will be adjusted at the same time, since
+          `mtime` is an attribute of the file.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
       - id: directory
         type: string
         brief: >
           Directory where the file is located. It should include the drive letter, when appropriate.
         stability: experimental
         examples: ['/home/user', 'C:\Program Files\MyApp']
+      - id: drive_letter
```

Comment:
network drives can have more than one letter, so I'd suggest a rename to `file.drive_name`, but also what's the use case for this attribute? Is it necessary at all?

Reply:
I cannot name you concrete usages for this particular field, but just as example to select all files located on this drive?
I'm ok to change name to be `drive_name`

Reply:
> I cannot name you concrete usages for this particular field, but just as example to select all files located on this drive?

Is this really useful? 

Perhaps we can remove this attribute altogether till there is a use case for it?